WordPress/gutenberg
# [RNMobile] Add custom matcher `toBeVisible`

## Description


Add custom matcher `toBeVisible` in order to provide more accurate assertions, in terms of component rendering, in integration tests.

**NOTE:** This custom matcher is in the process to be included in the `@testing-library/jest-native` package ([PR reference](https://github.com/testing-library/jest-native/pull/73)). Once it's included and a new version released, it's recommended to replace this implementation with the package.

## Testing Instructions




The matcher is only used for now in one test ([`packages/react-native-editor/src/test/index.test.js`](https://github.com/WordPress/gutenberg/blob/f02d823fc7101906976c4fcf61ad3b8b3c7d8c3f/packages/react-native-editor/src/test/index.test.js#L192)), so we'd only need to check that the test passes either by checking the PR checks (`Unit Tests / Mobile`) or running the command `npm run native test`.

## Screenshots 
N/A

## Types of changes




Enhancement

## Checklist:
